### PR TITLE
Implemented feature request #3139: Added cache multimap variants for the Rx API

### PR DESCRIPTION
**Pull request by mlkammer**
Hi @mrniko , I have added this PR to implement the feature request I reported in #3139. I tried my best to adhere to the existing code style. If you have the time to take a look at it, that would be great. Thank you.

Kind regards,
Marnix Kammer

### CODE REVIEW

File: redisson/src/main/java/org/redisson/api/RListMultimapCacheRx.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.redisson.api;
+
+/**
+ * Rx-ified version of {@link RListMultimapCache}.
+ *
+ * @author Nikita Koksharov
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/api/RListMultimapCacheRx.java:21]: please specify real author

File: redisson/src/main/java/org/redisson/api/RMultimapCacheRx.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+package org.redisson.api;
+
+import io.reactivex.Single;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Rx-ified version of {@link RMultimapCache}.
+ *
+ * @author Nikita Koksharov
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/api/RMultimapCacheRx.java:25]: please specify real author

File: redisson/src/main/java/org/redisson/api/RSetMultimapCacheRx.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.redisson.api;
+
+/**
+ * Rx-ified version of {@link RSetMultimapCache}.
+ *
+ * @author Nikita Koksharov
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/api/RSetMultimapCacheRx.java:21]: please specify real author

File: redisson/src/main/java/org/redisson/api/RedissonRxClient.java [hunk 4]
```diff
@@ -15,6 +15,7 @@
  */
 package org.redisson.api;
 
+import org.checkerframework.checker.units.qual.K;
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/api/RedissonRxClient.java:18]: this dependency should be removed

File: redisson/src/main/java/org/redisson/rx/RedissonListMultimapCacheRx.java [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+ */
+package org.redisson.rx;
+
+import org.redisson.RedissonList;
+import org.redisson.RedissonListMultimapCache;
+import org.redisson.api.RListRx;
+
+/**
+ *
+ * @author Nikita Koksharov
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/rx/RedissonListMultimapCacheRx.java:24]: please specify real author

File: redisson/src/main/java/org/redisson/rx/RedissonSetMultimapCacheRx.java [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+package org.redisson.rx;
+
+import org.redisson.RedissonSet;
+import org.redisson.RedissonSetMultimapCache;
+import org.redisson.api.RSetRx;
+import org.redisson.api.RedissonRxClient;
+
+/**
+ *
+ * @author Nikita Koksharov
```
Comment (mrniko) [inline redisson/src/main/java/org/redisson/rx/RedissonSetMultimapCacheRx.java:25]: please specify real author

File: redisson/src/main/java/org/redisson/api/RListMultimapCacheRx.java (hunk 1 above)
Comment (mrniko) [inline redisson/src/main/java/org/redisson/api/RListMultimapCacheRx.java:21]: please specify real author

**Review by mrniko — CHANGES_REQUESTED** [review]

File: redisson/src/main/java/org/redisson/api/RedissonRxClient.java (hunk 4 above)
Comment (mlkammer) [reply redisson/src/main/java/org/redisson/api/RedissonRxClient.java:18]: Very sharp, this was unintended, for some reason my IDE added this. Removed it now.

**Review by mrniko — APPROVED** [review]

**Comment by mrniko** [comment]
Thanks for contribution!

**Comment by mlkammer** [comment]
@mrniko Thanks for the quick review and approval/merge! You have done great work on this library, glad I could contribute some small thing.

**Pull request merged**